src/pages/docs/creators/python/guides/golem-images.md

```diff
@@ -0,0 +1,94 @@
+
+Any images that you intend to specify in your demand must be publicly available, so that the providers can download and deploy them.
+Therefore, you either need to publish them to our registry or host it yourself in a publicly-accessible location.
+
+Within our registry, the images can be identified by their image hash or their tag name.
+The type of identifier depends on the way you publish your image and is driven by intended usage.
+
+If you intend to use your image just for testing, it is enough to use the hashes and upload the images anonymously to the registry.
+On the other hand, if you intend to work on a more complex project, where you would like to use several different versions of your image or collaborate with other developers - you should consider creating an account in the registry and use tags to describe your images. Both cases are illustrated in our examples.
+
+- Publishing the image anonymously. ([example](/docs/creators/javascript/examples/tools/publishing-custom-images#publishing-custom-golem-image-to-the-registry-hash-based))
+- Publishing the image using tags. ([example](/docs/creators/javascript/examples/tools/publishing-custom-images#publishing-custom-golem-image-to-the-registry-tag-based))
+- Using the tag or hash in a requestor script. ([example](/docs/creators/javascript/examples/working-with-images))
+
+## Dockerfile command support 
+
+All the Docker commands that are related to the definition of the image content are supported. So if you can build a Docker image from your Dockerfile, you should be able to convert it to a Golem image.
+
+Please take into account the following points:
+
+### VOLUME
+
+If your application requires transferring files to and/or from the provider node, you'll need to specifically define a place (or places) in the container's file system that will be used for file transfers. These places are called volumes.
+
+Additionally, in the case of large files, it is recommended to generate and store them in directories defined using the VOLUME clause. In such cases, they are stored on the host's disk drive and do not consume the storage available in RAM.
+
+{% alert level="warning" %}
+
+When you define the image (in Dockerfile), do not copy your files into folders that are defined as volumes. When a Golem virtual machine is started, __a new directory__ is created in the host's file system for each of the defined volumes. This directory is then made available inside the VM under its specified path (for example: /golem/input).
+
+{% /alert %}
+
+### WORKDIR
+
+This will define the default directory to be used in shell commands sent to a remote computer once the VM is running.
+
+### ENTRYPOINT, CMD
+
+Because of how Golem's VM execution unit works, Docker's `ENTRYPOINT` and `CMD` statements are effectively ignored. You need to pass the relevant initialization commands as part of the task sent to a remote computer as a part of your task function or use the `beforeEach()` method. See examples.
```

Comment:
this `beforeEach` looks like something accidentally carried from the JS tutorial...